taskt/Core/Automation/Commands/Word Commands/WordAddDocumentCommand.cs

```diff
@@ -2,58 +2,60 @@
 using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
+using taskt.Core.Utilities.CommonUtilities;
 using taskt.UI.CustomControls;
 using taskt.UI.Forms;
-using taskt.Core.Utilities.CommonUtilities;
+using Application = Microsoft.Office.Interop.Word.Application;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("Word Commands")]
-    [Attributes.ClassAttributes.Description("This command adds a new Word Document.")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to add a new document to a Word Instance")]
-    [Attributes.ClassAttributes.ImplementationDescription("This command implements Excel Interop to achieve automation.")]
+    [Group("Word Commands")]
+    [Description("This command adds a new Word Document to a Word Instance.")]
+
     public class WordAddDocumentCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please Enter the instance name")]
-        [Attributes.PropertyAttributes.InputSpecification("Enter the unique instance name that was specified in the **Create Word** command")]
-        [Attributes.PropertyAttributes.SampleUsage("**myInstance** or **wordInstance**")]
-        [Attributes.PropertyAttributes.Remarks("Failure to enter the correct instance name or failure to first call **Create Word** command will cause an error")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        [PropertyDescription("Word Instance Name")]
+        [InputSpecification("Enter the unique instance that was specified in the **Create Application** command.")]
+        [SampleUsage("MyWordInstance || {vWordInstance}")]
+        [Remarks("Failure to enter the correct instance or failure to first call **Create Application** command will cause an error.")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
         public string v_InstanceName { get; set; }
 
         public WordAddDocumentCommand()
         {
-            this.CommandName = "AddDocumentCommand";
-            this.SelectionName = "Add Document";
-            this.CommandEnabled = true;
-            this.CustomRendering = true;
+            CommandName = "AddDocumentCommand";
+            SelectionName = "Add Document";
+            CommandEnabled = true;
+            CustomRendering = true;
         }
+
         public override void RunCommand(object sender)
         {
-            var engine = (Core.Automation.Engine.AutomationEngineInstance)sender;
+            var engine = (AutomationEngineInstance)sender;
             var vInstance = v_InstanceName.ConvertToUserVariable(engine);
             var excelObject = engine.GetAppInstance(vInstance);
```

Comment:
please change `excelObject` to `wordObject`.
Make sure to update its references in the code.

Reply:
done